Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "9625633055b34c0453607555f214afe555b625e9d155b9b0df0d1c0feea03954"
+            "sha256": "b4a28cc75471a3f803302901ba400840751ca1f4a6dd14b4e33f947558898e63"
```

Comment:
how are these changes made, e.g. where does this digest come from?

Reply:
This is how pipenv works. See https://pipenv.pypa.io/en/latest/.